runtime/src/main/java/org/corfudb/util/CFUtils.java

```diff
@@ -145,6 +150,37 @@ public static void runAfter(Duration duration, Runnable toRun) {
         return CompletableFuture.allOf(futuresArr);
     }
 
+    /**
+     * Takes a list of the completable futures and returns the CompletableFuture of a list.
+     *
+     * @param futures A list of completable futures, perhaps a result of a map function.
+     * @param <T>     A return type of the future.
+     * @return A completable future, which completes with a list of the results.
+     */
+    public static <T> CompletableFuture<List<T>> sequence(List<CompletableFuture<T>> futures) {
+        return CompletableFuture
```

Comment:
Since I already have implemented `allOf` in CfUtils, it had better to use it:
```
return allOf(futures).thenApply(x -> futures.stream()
                        .map(CompletableFuture::join)
                        .collect(Collectors.toList()));
```